R/execute_pipeline.R

```diff
@@ -376,6 +387,10 @@ run_targets_pipeline <- function(
     script = glue("{store}.R"),
     store = store,
     callr_function = NULL
```

Comment:
@myushen and @susansjy22 why callr_function is NULL? should this argument depend whether we are debugging (=NULL) or not (!= NULL)?

Thanks